app/src/main/java/com/example/busschedule/ui/BusScheduleScreens.kt

```diff
@@ -167,14 +191,23 @@ fun BusScheduleScreen(
         Row(
             modifier = Modifier
                 .fillMaxWidth()
-                .padding(dimensionResource(R.dimen.padding_medium)),
+                .padding(dimensionResource(R.dimen.padding_medium))
+                .padding(
+                    PaddingValues(
+                        top = contentPadding.calculateTopPadding(),
+                    )
+                ),
```

Comment:
This needs to be updated in the main branch as well. 

Could we simplify this code?
.padding(top = contentPadding.calculateTopPadding())

https://github.com/google-developer-training/basic-android-kotlin-compose-training-bus-schedule-app/pull/7#discussion_r1290843497